RELEASE_NOTES.md

```diff
@@ -4,6 +4,29 @@ This is a collection of release notes for the SL Design System. Each release not
 
 The release notes are ordered by the date the release was made. From latest, to oldest.
 
+# October 21, 2025
+
+Tree has been promoted from draft to preview.
+
+## New features
+- [`button`](https://github.com/sl-design-system/components/blob/main/packages/components/button/CHANGELOG.md) has new `fill` and `variant` properties that will apply to all buttons inside the button bar.
+- [`clickedu`](https://github.com/sl-design-system/components/blob/main/packages/themes/clickedu/CHANGELOG.md) has an updated color palette with new brand colors.
+- [`icon`](https://github.com/sl-design-system/components/blob/main/packages/components/icon/CHANGELOG.md) Support Font Awesome 7.1.
+- [`tree`](https://github.com/sl-design-system/components/blob/main/packages/components/tree/CHANGELOG.md) now support sorting
+- [`virtual-list`](https://github.com/sl-design-system/components/blob/main/packages/components/virtual-list/CHANGELOG.md) is a new utility package that provides a virtual scrolling solution based on `@tanstack/virtual-core`.
```

Comment:
Maybe it's worth to mention that `@lit-labs/virtualizer` is no longer necessary?